src/ngraph_backend_manager.h

```diff
@@ -47,25 +47,25 @@ struct Backend {
 class BackendManager {
  public:
   // Returns the backend name currently set
-  static string GetCurrentlySetBackendName() {
-    return BackendManager::ng_backend_name_;
-  };
+  // If env variable NGRAPH_TF_BACKEND is set it has precedence
```

Comment:
It has precedence only over the BackendManager and not the config, correct?

Reply:
Yes.